website/docs/r/vpn_server_configuration.html.markdown

```diff
@@ -152,16 +152,28 @@ A `ipsec_policy` block supports the following:
 
 A `radius_server` block supports the following:
 
-* `address` - (Required) The Address of the Radius Server.
+* `address` - (Optional) The Address of the Radius Server. Used to create single Radius Server configuration. Exists for backward compatibility purpose.
 
-* `secret` - (Required) The Secret used to communicate with the Radius Server.
+* `secret` - (Optional) The Secret used to communicate with the Radius Server. Used to crate single Radius Server configuration. Exists for backward compatibility purpose.
+
+* `server` - (Optional) One or more `server` blocks as defined below. Used to create multiple Radius Servers configuration.
 
 * `client_root_certificate` - (Optional) One or more `client_root_certificate` blocks as defined above.
 
 * `server_root_certificate` - (Optional) One or more `server_root_certificate` blocks as defined below.
 
 ---
 
+A `server` block supports the following:
+
+* `address` - (Required) The Address of the Radius Server.
+
+* `secret` - (Required) The Secret used to communicate with the Radius Server.
+
+* `score` - (Required) The score of the Radius Server determines the priority of the server. Ranges from 1 to 30.
```

Comment:
```suggestion
* `score` - (Required) The Score of the Radius Server determines the priority of the server. Ranges from 1 to 30.
```
Making it upper case to serve Case consistence as descriptions of `address` and `secret`.